### ISSUE TITLE
Error Running Docker Compose on RAMP Model

### ISSUE THREAD

**Issue by shaimaahegazy** [radiantearth/model_ramp_baseline#4]
Hello, 

I'm trying to run the RAMP model, but I face a problem when I run the following command:

 `docker` compose up model_ramp_baseline_v1_gpu

The error states:

> service "model_ramp_baseline_v1_cpu" refers to undefined volume "/data/input": invalid compose project.

Although I ran the command for gpu, the error states the cpu version, also I have set the INPUT_DATA variable, but not sure why I'm getting this error.  I am using a Windows environment.

Your help would be greatly appreciated.

Thanks,
Shaimaa

**Comment by guidorice** [comment]
@shaimaahegazy thanks for your inquiry. I believe the commands on Windows will vary depending on your environment and shell, etc. From the Ramp model readme:

> Please note: these command-line examples were tested on Linux and MacOS. Windows and WSL users may need to substitute appropriate commands, especially for setting environment variables.

For us to be able to assist further, would you describe more about your environment, like Windows PowerShell, or WSL, and what shell are you using (bash, zsh, other)?

**Comment by shaimaahegazy** [comment]
Thank you @guidorice for your reply. I'm actually using Windows PowerShell. I was able to solve my previous problem by appropriately setting the environment variable using "$env:INPUT_DATA=""" and "$env:OUTPUT_DATA=""".

I am also able to run the cpu model, but still trying to understand the results. However, I have a problem running the gpu version, where it says:

> Error response from daemon: Unknown runtime specified nvidia

Could you please help me figuring out this error.

Thanks a lot!

**Comment by guidorice** [comment]
@shaimaahegazy glad to hear you were able to run the cpu image for inferencing. I am surprised it worked without specifying a path for your INPUT_DATA and OUTPUT_DATA.

The output of the model is summarized in the model documentation here: 
https://github.com/radiantearth/model_ramp_baseline/blob/main/docs/index.md#structure-of-output-data

More background about the model is available from the model developers (DevGlobal and the Ramp project) here https://rampml.global/project-introduction/ .

> Error response from daemon: Unknown runtime specified nvidia

This error likely indicates that the NVIDIA container toolkit and CUDA drivers are not installed on your system. I do not actually know how to do that on Windows. Here is the guide I followed for Linux where I developed and tested the GPU inferencing docker image: https://docs.nvidia.com/ai-enterprise/deployment-guide/dg-docker.html

The results should be the same regardless of running on cpu or gpu, but the gpu runs faster.
